src/features/PathAutocompleteProvider.ts

```diff
@@ -293,6 +293,22 @@ export class PathAutocomplete implements vs.CompletionItemProvider {
         var items = [];
 
         Object.keys(configuration.data.pathMappings || {})
+            // if insertedPath is '@view/'
+            // and mappings is [{key: '@', ...}, {key: '@view', ...}]
+            // and it will match '@' and return wrong items { currentDir: 'xxx',  insertedPath: 'view/'}
+            // solution : Sort keys by matching longest prefix, and it will match key(@view) first
+            .sort((key1, key2) => {
+                const f1 = insertedPath.startsWith(key1)
```

Comment:
can we add semi-colons to the new code added here ?
we used them as part of this project

Reply:
i add it